README.md

```diff
@@ -1,3 +1,130 @@
 # You need a fahrschein to use the (nakadi event) bus
 
-## See `src/test/java/org/zalando/fahrschein/salesorder/Main.java` for an example how to use the library
+
+[![Build Status](https://travis-ci.org/zalando-incubator/fahrschein.svg?branch=master)](https://travis-ci.org/zalando-incubator/fahrschein)
+[![Release](https://img.shields.io/github/release/zalando-incubator/fahrschein.svg)](https://github.com/zalando-incubator/fahrschein/releases)
+[![Maven Central](https://img.shields.io/maven-central/v/org.zalando/fahrschein.svg)](https://maven-badges.herokuapp.com/maven-central/org.zalando/fahrschein)
+
+Fahrschein is a library for consuming events from a nakadi event bus.
+
+## Features
+
+ - Consistent error handling
+    - `IOException` handled as retryable
+    - `RuntimeException` aborts processing and can be handled outside the main loop
+ - No unnecessary buffering or line-based processing
+    - Less garbage and higher performance
+    - Better integration with Jackson json parser
+
+## Installation
+
+Fahrschein is available in maven central, so you only have to add the following dependency to your project:
+
+```xml
+<dependency>
+    <groupId>org.zalando</groupId>
+    <artifactId>fahrschein</artifactId>
+    <version>${fahrschein.version}</version>
+</dependency>
+```
+
+## Fahrschein compared to other nakadi client libraries
+
+|                      | Fahrschein                                                        | Nakadi-Klients        | Straw               |
+| -------------------- | ----------------------------------------------------------------- | --------------------- | ------------------- |
+| Dependencies         | Spring (http client and jdbc), Jackson, Postgres (optional)       | Scala, Akka, Jackson  | None                |
+| Cursor Management    | In-Memory / Persistent (Postgres)                                 | In-Memory             |                     |
+| Partition Management | In-Memory / Persistent (Postgres)                                 |                       |                     |
+| Error Handling       | Automatic reconnect with exponential backoff                      | Automatic reconnect   | No error handling   |
+
+## Usage
```

Comment:
I'd suggest to have a mini-minimal example in the same section as the `pom` fragment and down here multiple examples separated by feature (i.e. do not mix low-level and high-level api).